src/functionality.js

```diff
@@ -0,0 +1,162 @@
+import './styles.css';
+import { renderTodoList, tasks } from './index.js';
+
+const saveTasksToLocalStorage = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
```

Comment:
- [ ] There are some serious bugs in your app. kindly consider fixing the following bugs:
   - When I check the todos as completed and then press clear all button.the indexes gets updated but some unwanted todos also gets deleted. you can check the video for proof.
   - Whenever I refresh your app, the app behave weirdly. the edit functionally is gone after refresh.
video

https://github.com/uno36/To-do-list/assets/96683843/67890b00-265d-4a89-997b-57ab223e532e